tasks/requirements.yml

```diff
@@ -0,0 +1,7 @@
+---
+- name: Install pip requirements for home-assistant-supervised-ansible
+  pip:
```

Comment:
Since you are building this new, might want to use fully qualified module names now that collections are a thing?

`ansible.builtin.pip`

Reply:
That reminds me that I meant to add ansible-lint to Github actions workflow.